doc/network_chaos.md

```diff
@@ -116,3 +116,24 @@ A Network Corrupt action causes packet corruption. To add a Network Corrupt acti
 >```
 >
 > **corrupt** specifies the percentage of packet corruption.
+
+## Network Bandwidth Action
+
+Network Bandwidth Action is used to limit the network bandwidth. To add a Network Bandwidth Action, locale and edit the corresponding template in [/examples](../examples/network-bandwidth-example.yaml).
+
+Right now *minikube* doesn't support this feature as `CONFIG_NET_SCH_TBF` was disabled in minikube's image.
+> In this case, three action specific attributes are required - rate, buffer and limit.
+>
+>```yaml
+> bandwidth:
+>   rate: 10 kbps
+>   buffer: 1000
+>   limit: 100
+>```
+>
+> **rate** allows bps, kbps, mbps, gbps, tbps unit. bps means bytes per second.
+> **limit** defines the number of bytes that can be queued waiting for tokens to become available.
+> **buffer** is the maximum amount of bytes that tokens can be available for instantaneously.
+> **peakrate** is the maximum depletion rate of the bucket.
```

Comment:
Maybe we should add more docs here ? For me, `tc-tbf` is something new. It's quite difficult to understand `peakrate` and `minburst`.

Reply:
I'm totally agree, even further maybe we can consider to only expose small set configuration like rate, and hide the complexity behind it, the user use `chaos-mesh` doesn't necessarily to learn the details about tbf.